### DISCUSSION TITLE
Where we can find 'Atomic Lake'?

### DISCUSSION THREAD

**Discussion by SANKARAMDAS** [ParanoidUser/yolo#823] [Galaxy Trivia] [answered]
When it was created? Radiation Level nowadays?

**Comment by hasanerzi** [comment, marked as answer]
Lake Chagan or Lake Shagan also known as Lake Balapan, is a in Kazakhstan created by the on January 15, 1965 (Chagan nuclear test). 
The water continues to be radioactive - about 100 times more than the permitted level of radionuclides in the water.

**Reply by ParanoidUser** [reply]
Congratulations! 👏👏 You have just earned the prestigious [🥉 Galaxy Brain](https://github.com/hasanerzi?tab=achievements&achievement=galaxy-brain) badge. Keep up the great work and unlock the Silver 🥈 badge next by answering 16 questions. Don't forget to post your own questions too! 😊🎉
